doc/IMPLEMENTING_NEW_OP.md

```diff
@@ -6,16 +6,17 @@ Customize op can also be implemented in similar way.
 
 ### Backend
 
-1.  Find specification from [onnx/Operators](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
-2.  Decide if need a specific handler.  
+1.  Verify the latest master version of ONNX is installed on your environment
+2.  Find specification from [onnx/Operators](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
+3.  Decide if need a specific handler.
     Op doesn't need a specific handler when tf's specification highly matches onnx's, means:
     
     - inputs are same
     - tf's attributes is a subset of onnx's    
     - attr doesn't exist in tf could be set by `DEFAULT_ONNX_ATTR_PER_OP` in `backend.py`  
     
     otherwise, op needs a specific handler.
-3.  Implement. All inputs and attrs could get from step 1.
+4.  Implement. All inputs and attrs could get from step 1.
```

Comment:
This should be changed to refer to step 2.

Reply:
Done, @chinhuang007 please review again. Thanks